packages/coil-extension/src/background/services/Stream.ts

```diff
@@ -323,7 +323,9 @@ class StreamAttempt {
     this._connection = await createConnection({
       ...this._spspDetails,
       plugin,
-      slippage: 0.05
+      slippage: 1.0,
```

Comment:
We will want to do this for the oauth-scripts too I assume

Reply:
@sublimator updated. What's the best way to test oauth-scripts locally (pref. against staging)?